### ISSUE TITLE
RoutingBGP throwing error Cannot read property 'toUpperCase' of undefined

### ISSUE THREAD

**Issue by melkypie** [F5Networks/f5-declarative-onboarding#249]
### Environment
 * Declarative Onboarding Version: 1.22.0
 * BIG-IP Version: 15.1.3 Build 0.0.11 Final

### Summary

Been trying to setup CIS automatically using DO but I can't get RoutingBGP to configure, it is throwing error `Cannot read property 'toUpperCase' of undefined`.

### Steps To Reproduce
1. Submit the following declaration:
```json
{
    "schemaVersion": "1.22.0",
    "class": "Device",
    "async": true,
    "label": "Cloudinit Onboarding",
    "Common": {
        "class": "Tenant",
        "hostname": "hostname",
        "rd0": {
            "class": "RouteDomain",
            "id": 0,
            "routingProtocols": [
                "BGP"
            ]
        },
        "k8scalico": {
            "class": "RoutingBGP",
            "localAS": 64512,
            "peerGroups": [
                {
                    "name": "k8s-group",
                    "remoteAS": 64512
                }
            ],
            "neighbors": [
                {
                    "address": "10.1.1.2",
                    "peerGroup": "k8s-group"
                },
                {
                    "address": "10.1.1.3",
                    "peerGroup": "k8s-group"
                },
                {
                    "address": "10.1.1.4",
                    "peerGroup": "k8s-group"
                }
            ]
        }
    }
}
```

2. Observe the following error response:
```json
[
    {
        "id": "000000-0000-0000-0000-000000000000",
        "selfLink": "https://localhost/mgmt/shared/declarative-onboarding/task/000000-0000-0000-0000-000000000000",
        "code": 500,
        "status": "ERROR",
        "message": "invalid config - rolled back",
        "errors": [
            "Cannot read property 'toUpperCase' of undefined",
            "Cannot read property 'toUpperCase' of undefined"
        ],
        "result": {
            "class": "Result",
            "code": 500,
            "status": "ERROR",
            "message": "invalid config - rolled back",
            "errors": [
                "Cannot read property 'toUpperCase' of undefined",
                "Cannot read property 'toUpperCase' of undefined"
            ]
        },
    "declaration": {}
}
```

### Expected Behavior
For BGP to be configured.

### Actual Behavior

Throws the above error

**Comment by dstokesf5** [comment]
Thank you for reporting this. I have confirmed the bug and added it to our internal product backlog as AUTOTOOL-2616.

**Comment by dstokesf5** [comment]
This issue has been resolved with the 1.23.0 release of Declarative Onboarding.
